test/integration/run_envoy_test.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+source "$TEST_RUNDIR/test/integration/test_utility.sh"
+
+start_test Launching envoy with a bogus command line flag.
+${ENVOY_BIN} --bogus-flag
+EXIT_CODE=$?
+# The test should fail if the Envoy binary exits with anything other than 1.
+if [[ $EXIT_CODE -ne 1 ]]; then
+    echo "Envoy exited with code: ${EXIT_CODE}"
+    exit 1
+fi
+
+start_test Launching envoy with no args starts a server properly.
```

Comment:
I actually think this is a regression and you found a bug that crept in at some point. We used to require `-c` to specify a config. Running Envoy without some config doesn't make any sense IMO. I'm guessing this crept in when we added options to specify YAML config directly on the CLI or something like that. I think if config hasn't been supplied somehow it should be an error. Can you take a look?

Reply:
np -- I think you're right; should be easy.

/wait